lib/router/transition-intent/named-transition-intent.ts

```diff
@@ -222,13 +223,16 @@ export default class NamedTransitionIntent<T extends Route> extends TransitionIn
         if (oldParams.hasOwnProperty(paramName)) {
           params[paramName] = oldParams[paramName];
         } else {
-          throw new Error(
-            "You didn't provide enough string/numeric parameters to satisfy all of the dynamic segments for route " +
-              name
-          );
+          missingParams.push(paramName);
         }
       }
     }
+    if (missingParams.length > 0) {
+      throw new Error(
+        "You didn't provide enough string/numeric parameters to satisfy all of the dynamic segments for route ${name}." +
+          ' Missing params: ${missingParams}'
```

Comment:
unfortunately linter thinks it's a good idea to put this line with single quotes even though previous one is double quoted.